perf-tool/src/monitor.cpp

```diff
@@ -214,3 +214,136 @@ JNIEXPORT void JNICALL MonitorContendedEntered(jvmtiEnv *jvmtiEnv, JNIEnv *env,
         }
     }
 }
+
+JNIEXPORT void JNICALL MonitorContendedEnter(jvmtiEnv *jvmtiEnv, JNIEnv *env, jthread thread, jobject object){
+	json j;
+	auto jMethods1 = json::array();
+	json jMethod1;
+	jvmtiError error;
+	jvmtiMonitorUsage monitor_usage;
+	error = jvmtiEnv->GetObjectMonitorUsage(object, &monitor_usage);
+	if(check_jvmti_error(jvmtiEnv, error, "Unable to get Monitor usage info."))
```

Comment:
Please use the same indentation style as in the rest of the files (4 spaces) and add one space after `if` keywork